hypertrace-trace-enricher-impl/src/test/java/org/hypertrace/traceenricher/enrichment/enrichers/ApiBoundaryTypeAttributeEnricherTest.java

```diff
@@ -117,6 +120,52 @@ public void test_enrichEvent_loopToItsOwnService_findTwoAPIEntries() {
     Assertions.assertEquals(EnrichedSpanUtils.getApiBoundaryType(secondExit), Constants.getEnrichedSpanConstant(EXIT));
   }
 
+  @Test
```

Comment:
https://codecov.io/gh/hypertrace/hypertrace-trace-enricher/pull/26/diff did you test negative scenarios?

Reply:
Added now.